packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -4121,7 +4121,7 @@ export class ITxn extends Op {
 	execute(stack: TEALStack): void {
 		if (this.interpreter.innerTxnGroups.length === 0) {
 			throw new RuntimeError(RUNTIME_ERRORS.TEAL.NO_INNER_TRANSACTION_AVAILABLE, {
-				version: this.interpreter.tealVersion,
```

Comment:
why renaming?

Reply:
Oh! Its not match with message map in ERROR. So I fix it to same with other.